APMrover2/Log.cpp

```diff
@@ -471,7 +525,10 @@ void Rover::Log_Write_Startup(uint8_t type) {}
 void Rover::Log_Write_Current() {}
 void Rover::Log_Write_Nav_Tuning() {}
 void Rover::Log_Write_Performance() {}
-int8_t Rover::process_logs(uint8_t argc, const Menu::arg *argv) { return 0; }
+int8_t Rover::process_logs(uint8_t argc, const Menu::arg *argv)
+{
+    return 0;
+}
```

Comment:
I think this is one of the exceptions where we allow it to be on the same line (and it looks much better that way).